test/util/pki.go

```diff
@@ -0,0 +1,91 @@
+package util
+
+import (
+	"context"
+	"crypto/rand"
+	"crypto/rsa"
+	"crypto/x509"
+	"crypto/x509/pkix"
+	"encoding/pem"
+	"fmt"
+	"math/big"
+	"net"
+	"time"
+)
+
+// GenerateTestCertificate creates a test certificate with the given DNS names and IP addresses
+func GenerateTestCertificate(ctx context.Context, dnsNames []string, ipAddresses []string, duration time.Duration) ([]byte, []byte, error) {
```

Comment:
We already have code to do this here:
https://github.com/openshift/hypershift/blob/main/support/certs/tls.go

Reply:
solved 👍